<launch>
  <include file="$(find tesbot_gazebo)/launch/tesbot.launch">
    <param name="gui" value="false" />
  </include>
  <node pkg="tesbot_keyboard" type="tesbot_keyboard_node" name="tesbot_keyboard_node" output="screen">
    <param name="model_path" type="str" value="$(find tesbot_description)/urdf/tesbot.urdf" />
    <param name="sleep_time" type="int" value="5" />
  </node>
</launch>
